docs/build-for-developers/cli-usage.md

```diff
@@ -1,98 +1,137 @@
 ---
 title: Using the CLI
-sidebar_label: CLI Usage
+sidebar_label: Basic Usage
 slug: /cli-usage
 ---
 
-## Prerequisites
+You're probably here to run jobs (expressions) or workflows, which the CLI makes
+easy. Here's an outline of the basic usage scenarios
```

Comment:
Replace the second sentence with something like "Keep reading for an outline of the basic usage scenarios of the CLI."